### DISCUSSION TITLE
抢到了，牛逼!

### DISCUSSION THREAD

**Discussion by drinknaixi-1** [mikumifa/biliTickerBuy#207] [聊天专区]
![IMG_20240629_120719](https://github.com/mikumifa/biliTickerBuy/assets/173991673/ba50735e-4fe7-4922-a328-fdf38fe7c072)

**Comment by Sduipd** [comment]
牛逼，怎么做到的，我开了没抢到

**Reply by drinknaixi-1** [reply]
按照作者说的做的

**Comment by Kocurrrrrr** [comment]
我也抢到了，爽！

**Comment by Asuna380** [comment]
你开的自动验证还是手动验证啊，我开的手动

**Reply by drinknaixi-1** [reply]
自动

**Comment by mikumifa** [comment]
作者自己没抢到🥲

**Reply by Asuna380** [reply]
大佬，请求那个geetset失败就是没抢到而不是配置啥错了吧

**Reply by XINGYI0309** [reply]
还有二战，会赢的（awa）

**Reply by mikumifa** [reply]
不好说

**Reply by akie001** [reply]
是不是参数太严谨了，我间隔300抢到了

**Reply by PxxCharon** [reply]
> 是不是参数太严谨了，我间隔300抢到了

你是用的rrocr的验证吗？

**Reply by akie001** [reply]
> > 是不是参数太严谨了，我间隔300抢到了
> 
> 你是用的rrocr的验证吗？

本地自动验证

**Reply by PxxCharon** [reply]
> > > 是不是参数太严谨了，我间隔300抢到了
> > 
> > 
> > 你是用的rrocr的验证吗？
> 
> 本地自动验证

是capsolver吗？

**Comment by alphardex** [comment]
我手动也没抢到😭
![Snipaste_2024-06-29_12-09-24](https://github.com/mikumifa/biliTickerBuy/assets/22198352/d57518ab-9537-4525-a51a-d60c4e6b0130)

**Comment by 284808515** [comment]
我也抢到了，大佬牛批

**Comment by fyydvay** [comment]
我也是，30s抢到VIP

**Comment by sawslsl** [comment]
开的哪个验证啊，还有间隔多少，大佬

**Reply by drinknaixi-1** [reply]
rrocr，间隔300

**Reply by sawslsl** [reply]
那个api-key是什么，要填吗

**Reply by drinknaixi-1** [reply]
花十块钱去买积分，然后把key复制过来

**Reply by 1093201139** [reply]
![{F8327A81-94CD-43a0-A693-A318261FC19D}](https://github.com/mikumifa/biliTickerBuy/assets/58314404/0f306529-67bd-4e85-a662-476a45404ee2)
是这个东西吗

**Reply by alphardex** [reply]
用这个识别后底下validate是空的，测试结果里是验证码过期，算成功了吗
![c4e39c30-fb84-473e-a28e-576e2bc5b390](https://github.com/mikumifa/biliTickerBuy/assets/22198352/3547bba8-78dd-49f2-ad2b-91c46c1c7fdb)

**Reply by drinknaixi-1** [reply]
> ![{F8327A81-94CD-43a0-A693-A318261FC19D}](https://private-user-images.githubusercontent.com/58314404/344367979-0f306529-67bd-4e85-a662-476a45404ee2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2MzkzNDgsIm5iZiI6MTcxOTYzOTA0OCwicGF0aCI6Ii81ODMxNDQwNC8zNDQzNjc5NzktMGYzMDY1MjktNjdiZC00ZTg1LWE2NjItNDc2YTQ1NDA0ZWUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI5VDA1MzA0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM3NWQxMmQ2N2FhZWIwYjlhNzg4M2Q1MTI5ZDcyZDE2ZWY2MWJjZjkzODI2ZGQ1ODNkMTllMGMzYmU4MmI5MmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.m1aj4Lv46FllSRquJfHZlKFNYs8BM8aIGAncJ0-1czc) 这是什么东西？

是的

**Reply by IYngSnowSL** [reply]
> 花十块钱去买积分，然后把key复制过来

这个在哪里买啊，大佬求教

**Reply by drinknaixi-1** [reply]
> 用这个识别后底下validate是空的，测试结果里是验证码过期，算成功了吗 ![c4e39c30-fb84-473e-a28e-576e2bc5b390](https://private-user-images.githubusercontent.com/22198352/344368721-3547bba8-78dd-49f2-ad2b-91c46c1c7fdb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2Mzk0MzgsIm5iZiI6MTcxOTYzOTEzOCwicGF0aCI6Ii8yMjE5ODM1Mi8zNDQzNjg3MjEtMzU0N2JiYTgtNzhkZC00OWYyLWFkMmItOTFjNDZjMWM3ZmRiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI5VDA1MzIxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIzMmQwMTJjOTcxY2VlNWIzZTFhY2U5ZGVjNGFkMWJkYjEwYWYxNTQ5Zjk3NmYxZjY5MjQxYjM1MDQzNjZkMmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4gVYhautp5hroPpNBuOtebJeT8KmFH4R-zta_JTaTZc)

验证码过期就是成功了

**Reply by alphardex** [reply]
好，晚上就试试这个

**Reply by drinknaixi-1** [reply]
> > 花十块钱去买积分，然后把key复制过来
> 
> 这个在哪里买啊，大佬求教

网站的表格里不是有吗

**Comment by CtrlScrollScroll** [comment]
本地过验证码居然报错了
![5M0J0H64 (P_6I7QL9A43(4](https://github.com/mikumifa/biliTickerBuy/assets/44940408/55950963-f835-4c82-a875-5937377aef8f)

**Reply by mikumifa** [reply]
1%概率

**Reply by sawslsl** [reply]
那个amortor是怎么用的啊作者大大

**Comment by MegumiN152** [comment]
脚本报错，但是手机抢到了

**Comment by a-hard-leaner-bala** [comment]
抢到两张

**Comment by lllllis-02** [comment]
呜呜呜呜呜13,14一天也没抢到啊

**Comment by nikoyoke1** [comment]
想问问这个脚本是锁票，还是到时候要在web界面直接付款的

**Reply by nikoyoke1** [reply]
还有账号未实名认证可以抢吗

**Comment by IYngSnowSL** [comment]
寄咯

**Comment by SakatsukiAoi** [comment]
抢到了，感谢大佬

**Comment by alphardex** [comment]
尽管脚本没抢到，但还是用朋友的华为机抢到了，依旧感谢大佬🙇‍

**Comment by sppi8** [comment]
抢到了大佬nb

**Comment by keleus** [comment]
难绷12 14，一头一尾抢到了，13号药丸

**Reply by stukt18** [reply]
难绷就抢到13号vip

**Reply by stukt18** [reply]
想问问 用的大会员提前购还是普通的配置

**Reply by keleus** [reply]
@stukt18 我12号是手动的大会员提前，然后14号是18点抢到的普票。13号我也是抢的vip，但因为帮朋友一起，多人的就没抢到。单人的2个都抢到了

**Reply by stukt18** [reply]
@keleus 我12 14没看直接填的大会员提前的配置直接似了

**Comment by JieKBK** [comment]
抢到了，感谢每一位作者大大
